circle.yml

```diff
@@ -28,13 +28,14 @@ dependencies:
     - sudo chmod +x /usr/bin/docker
     # Override host go
     - sudo service docker start
-    - docker login -e $DOCKER_EMAIL -u $DOCKER_USER -p $DOCKER_PASS quay.io
+    # NOTE [ben]: @afdudley, this is a temporary hack to enable PRs as of 0.12.0
+    # - docker login -e $DOCKER_EMAIL -u $DOCKER_USER -p $DOCKER_PASS quay.io
```

Comment:
@AFDudley just to keep you in the loop; I've gone ahead and "done the stupid thing here", just to get it out of my way, but happy to work with you to get a clean decoupling of CI and CD in place.